appcues/src/main/java/com/appcues/trait/appcues/TooltipTrait.kt

```diff
@@ -70,10 +74,9 @@ internal class TooltipTrait(
 
         const val TYPE = "@appcues/tooltip"
 
-        val SCREEN_HORIZONTAL_PADDING = 12.dp
-        val SCREEN_VERTICAL_PADDING = 24.dp
-        private val MAX_WIDTH_DP = 350.dp
```

Comment:
do we need a max width and min height for consistency with https://github.com/appcues/appcues-ios-sdk/blob/traits-2.0/Sources/AppcuesKit/Presentation/Traits/Appcues/TooltipWrapperView.swift#L21-L22